test/e2e/specs/specs-playwright/wp-notifications__trash-spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+	setupHooks,
+	BrowserManager,
+	DataHelper,
+	LoginFlow,
+	PublishedPostsListPage,
+	CommentsComponent,
+	NavbarComponent,
+	NotificationsComponent,
+} from '@automattic/calypso-e2e';
+
+describe( DataHelper.createSuiteTitle( 'Notifications' ), function () {
```

Comment:
Equivalent test in Selenium: https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/specs-calypso/wp-notifications-spec.js